### DISCUSSION TITLE
Reolink Doorbell Wifi 2 Way Audio Not Working on the Newest Firmware (v3.0.0.3215)

### DISCUSSION THREAD

**Discussion by yzxchn** [koush/scrypted#1428] [Q&A] [answered]
I couldn't get 2 way audio working on the newest firmware (v3.0.0.3215), but it works if I downgrade to the previous one (v3.0.0.2033). Has anyone encountered the same, or been able to get it working on the latest firmware?

I'm using the Reolink plugin with Homekit integration.

_P.S. I have to use the newest firmware because I somehow couldn't get the Chime pairing working on the old one._

**Comment by koush** [comment, marked as answer]
known issue with their firmware.

**Comment by CFA2008** [comment]
So is this fixed? I’m on the newer software and 2 way audio doesn’t work still? Do I need to downgrade to a specific version? Or am I doing something wrong here? I see it trying to work in the logs when I press the 2 way audio button but it doesn’t activate the blue light or anything.
